std/experimental/allocator/building_blocks/allocator_list.d

```diff
@@ -1328,6 +1328,7 @@ template SharedAllocatorList(alias factoryFunction,
     assert(a.deallocateAll());
 }
 
+version (FreeBSD) {} else
```

Comment:
put some sort of comment marker here? a `//FIXME:` or `//BUG:` or something?
also a brief explanation of why it is disabled?